docs/vsphere-golden-image.md

```diff
@@ -144,8 +144,15 @@ To deploy the Windows VM as a reusable image, you have to first generalize the V
 such as installed drivers. Running the `sysprep` command with a *unattend* answer file generalizes the image and 
 makes it ready for future deployments, maintaining all the changes needed for the WMCO installation. 
 
-You can find an example [unattend.xml](unattend.xml) file in this repository. It is just an example and, it **should 
-not** be used directly. You must customize the example to fit your needs.
+A reference [unattend.xml](unattend.xml) file is provided, it **cannot** be used directly. You must customize it
+replacing the `WindowsPassword` parameter with the desired password for the Administrator user. The Password must
+meet complexity requirements as per [Microsoft documentation](https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/password-must-meet-complexity-requirements).
+
+Please note that, the value of the `ComputerName` property in the [unattend.xml](unattend.xml) file must follows the
+[Kubernetes' names specification](https://kubernetes.io/docs/concepts/overview/working-with-objects/names). The latter,
+also applies to Guest OS customization performed on the resulting template while creating new VMs.
+
+Before executing the `sysprep` command, clone the VM and then run `sysprep` command in the cloned VM.
```

Comment:
Please move this to the same line as `To execute the `sysprep` command use:` following the 120 column constraint.